Replicate missing package, device, and unit column headings

mlopezja opened the pull request:
Added some logic to fill in package, device, and unit column headings wherever they were missing.

mlopezja committed:
Replicate missing package, device, and unit column headings

blackk-foxx commented on csv_viewer.py:
```diff
@@ -17,8 +17,22 @@ def read_data(self, the_file):
+    @staticmethod
+    def fill_in_column_headers(rows):
+        number_of_header_rows = 3
+        for row_index, row in enumerate(rows):
+            if row_index == number_of_header_rows:
+                break
```
```suggestion
        for row in rows[:number_of_header_rows]:
```

mlopezja replied:
If we intend to use this viewer exclusively for the emon_results.csv files, I think it's reasonable to crash. Besides, these files rarely change format, so crashing may serve us as an indicator of some format error in the EMON output (or a corruption of the CSV file). Additionally, your suggestion seems more Pythonic, I like it.

blackk-foxx reviewed (commented):
One possible weakness in my suggestion is that it will crash in the rare case where there are less than 3 rows in the input file. But maybe crashing isn't unreasonable in that case.  Your thoughts?

mlopezja committed:
Apply review suggestion

Use slicing to obtain the list of rows to iterate inside the function that shows the table headers across all columns.

Co-authored-by: Todd Schwartz <barbarosa0620@gmail.com>
